aarch64-options.mk

```diff
@@ -0,0 +1,13 @@
+#
```

Comment:
For consistency with the runtime, I think it would be better to create this as `arch/aarch64-options.mk`.